sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClientOptions.cs

```diff
@@ -7,14 +7,14 @@
 namespace Azure.Security.KeyVault.Certificates
 {
     /// <summary>
-    /// Options that allow to configure the management of the request sent to Key Vault
+    /// Options that allow to configure the management of the request sent to Key Vault.
```

Comment:
Awkward sentence.  Allow who?
Maybe: "Options allowing configuration of requests sent to the Key Vault service."  ?